rest-api-model-server/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+plugins {
+    kotlin("jvm")
+    kotlin("plugin.allopen") version "1.7.20"
+    id("io.quarkus")
+    id("org.openapi.generator") version "6.2.1"
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+val quarkusPlatformGroupId: String by project
+val quarkusPlatformArtifactId: String by project
+val quarkusPlatformVersion: String by project
+val ktor_version: String by project
+val api_gen_version: String by project
+
+dependencies {
+    implementation(enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}"))
+    implementation("io.quarkus:quarkus-resteasy-reactive")
+    implementation("io.quarkus:quarkus-kotlin")
+    implementation("io.quarkus:quarkus-resteasy-reactive-jackson")
+    implementation("io.quarkus:quarkus-smallrye-openapi")
+    implementation("io.ktor:ktor-client-core:$ktor_version")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("io.quarkus:quarkus-arc")
+    implementation("org.modelix:model-client:1.3.2")
+    implementation("org.modelix.mps.api-gen:runtime:$api_gen_version")
+    implementation(project(":mps:solutions:University.Schedule.api"))
+    testImplementation("io.quarkus:quarkus-junit5")
+    testImplementation("io.rest-assured:rest-assured")
+}
+
+group = "org.modelix.sample"
+version = "1.0.0-SNAPSHOT"
+
+val openApiFile = layout.projectDirectory.file("../openapi/openapi.yaml")
```

Comment:
Can we move it to the parent? We will need this in the `rest-api-json-bulk` project as well.

Reply:
Maybe this can be done when the code generation in the JSON bulk API is is automated?